src/content/about/history.mdx

```diff
@@ -0,0 +1,27 @@
+import { IllustrationTextSection } from "../../components/sections/illustrationTextSection.js";
+import { MaxWidthWrapper } from "../../components/layout/maxWidthWrapper";
+import about1 from "../../images/about/Europa-center.png";
+import about2 from "../../images/about/streamline.png";
+
+<MaxWidthWrapper backgroundColor="#FFFFFF" gutterBottom>
```

Comment:
last minor comment! we can safely remove any background color that is #FFFFFF (white). we only need to define the background color for the sections that have that light grey background. you should be able to remove`backgroundColor="#FFFFFF"` here, as well as in `eduhex/eduHelxAbout.mdx`, `eduhelx/eduHelxReadMore.mdx`, and `home/homeIllustrations.mdx`